images/linux/scripts/installers/swift.sh

```diff
@@ -15,6 +15,19 @@ swift_tar_name="swift-$swift_version-RELEASE-ubuntu$image_label.tar.gz"
 swift_tar_url="https://swift.org/builds/swift-$swift_version-release/ubuntu${image_label//./}/swift-$swift_version-RELEASE/$swift_tar_name"
 download_with_retries $swift_tar_url "/tmp" "$swift_tar_name"
 
+# Verifing pgp signature using official swift pgp key. Referring to https://www.swift.org/install/linux/#Installation-via-Tarball
```

Comment:
typo should be `gpg`

Reply:
@samvantran I thought "gpg" it is OpenPGP encryption and signing tool. Also here is part of public key:
```
-----BEGIN PGP PUBLIC KEY BLOCK-----
Comment: 14F2 6682 D091 6CDD 81E3  7B6D 61B7 B526 D98F 0353

xsFNBFWpQAQBEAC+9wVlwGLy8ILCybLesuB3KkHHK+Yt1F1PJaI30X448ttGzxCz
PQ......
```

Reply:
yeah I realized my mistake when I went to read the swift docs (thanks for including the link in the comment)